apps/design-system/src/subjects/views/project-create/project-create-view.tsx

```diff
@@ -0,0 +1,20 @@
+import { noop, useTranslationStore } from '@utils/viewUtils'
+
+import { CreateProjectPage } from '@harnessio/ui/views'
+
+export const CreateProjectView = ({ isAdditional = false }: { isAdditional?: boolean }) => {
+  const commonProps = {
+    isLoading: false,
+    error: '',
+    onFormSubmit: noop,
+    useTranslationStore: useTranslationStore
+  }
+
+  return (
+    <>
+      {isAdditional && <CreateProjectPage {...commonProps} isAdditional={isAdditional} backLinkProps={{ to: '/' }} />}
+
+      {!isAdditional && <CreateProjectPage {...commonProps} logoutLinkProps={{ to: '/' }} />}
+    </>
+  )
```

Comment:
Not an issue, but sometimes it's easier and more readable to do logic outside of JSX.
```suggestion
  if (isAdditional) {
    return <CreateProjectPage {...commonProps} isAdditional={isAdditional} backLinkProps={{ to: '/' }} />
  }
  
  return <CreateProjectPage {...commonProps} logoutLinkProps={{ to: '/' }} />
```